AWSPinpoint/AWSPinpointNotificationManager.h

```diff
@@ -44,51 +44,111 @@ typedef NS_ENUM(NSInteger, AWSPinpointPushEventSourceType) {
 #pragma mark - Notification Helpers
 /**
  Returns a Boolean indicating whether the app is currently registered for remote notifications.
- @return BOOL YES if the app is registered for remote notifications and received its device token or NO if registration has not occurred, has failed, or has been denied by the user.
+ @return BOOL YES if the app is registered for remote notifications and received its device token or NO if registration
+ has not occurred, has failed, or has been denied by the user.
  */
 + (BOOL)isNotificationEnabled;
 
 #pragma mark - Interceptors
 /**
- Intercepts the `- application:didFinishLaunchingWithOptions:` application delegate.
+ Invoke this method from the `- application:didFinishLaunchingWithOptions:` application delegate method.
  
- Targeting must intercept this callback in order to report campaign analytics correctly.
+ The Pinpoint targeting client must intercept this callback in order to report campaign analytics correctly.
  
- @param launchOptions A dictionary indicating the reason the app was launched (if any). The contents of this dictionary may be empty in situations where the user launched the app directly. For information about the possible keys in this dictionary and how to handle them.
+ @param launchOptions A dictionary indicating the reason the app was launched (if any). The contents of this dictionary
+ may be empty in situations where the user launched the app directly. For information about the possible keys in this
```

Comment:
"For information about the possible keys in this
 dictionary and how to handle them."

Remove the fragment, since we would just be referring folks out to iOS documentation.